contributing-a-devnote.md

```diff
@@ -250,23 +351,95 @@ The first two settings, `jupyter: true` and `format: meca` should not be changed
+
+:::
+
+By now, you will have cloned the repo and made your own folder in the `dev-notes/*` folder, where you've been working so far.
+
+It's important to make sure that any changes that you've made in the repo are only to the contents of your new folder, and nowhere outside of that folder. You can check this by running `git status` and making sure all changes or additions are only in your DevNote folder. If you have changes outside you'll need to revert them before submitting.
+
+Once that's the case then go ahead and just locally commit your changes to the main branch:
+
+```sh
+git add .
+git commit -am "my commit message"
+```
+
+Next we're going to use the GitHub CLI to open a PR and in the process create our own fork of the DevNotes repo in our account. To do this, just run the following command from anywhere within the repository and follow the instructions:
+
+```sh
+gh pr create
+```
+
+The GitHub CLI will take you through the process of deciding to create a fork, creating the fork in your own account, pushing your changes to your fork and then opening the PR.
+
+At the end of the process, a link to your PR should be displayed, click that link to visit the PR on the central DevNotes repository.
+
+:::{tip} If you are not using the GitHub CLI
+:class: dropdown
+Please check the GitHub documentation on how to create a fork (https://docs.github.com/en/get-started/quickstart/fork-a-repo) and how to open a PR against an upstream repo (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+:::
+
+Once you've landed on the pull request page on GitHub and you're happy with the title and initial description look for a comment from the Curvenote preview bot it should look something like this.
+
+:::{figure} ./github-submit-bot.png
+The Curvenote Preview bot on GitHub.
+:::
+
+The comment from the Curvenote Bot should appear shortly after you've posted your PR and every time you push a new change to your branch, the PR will automatically update and the Curvenote Preview will automatically regenerate, this will including re-running any checks that have been configured to run.
 
-### Submitting your first draft (new docs)
+## Requesting review
 
-TODO:
+The review process is carried out over GitHub and the exact same PR that you just opened. Yourself, the editors and reviewers can all see the source code and access the preview links from the Cursor bot in order to guide the review and you may have already been contacted by one of the team when you opened the PR.
 
-1.  previewing (existing / new docs?)
-1.  the checks system (existing / new docs?)
-1.  troubleshooting (existing / new docs?)
+If you haven't just mention `@antonrmolina` in a comment and say this is ready for review.
```

Comment:
@antonrmolina I added this as a first step but really it's up to you what type of process you want to bring in here. There is the potential for asking people to create a PR that's marked as draft to start with. I'm not sure if checks will run on that though, that's something you could find out. But obviously there could be some more process here that you put in place in the documentation depending on how you want to deal with this. But this is a good start.